### ISSUE TITLE
Some KA-50 commands/buttons/switches are not exposed

### ISSUE THREAD

**Issue by truhlikfredy** [dcs-bios/dcs-bios#197]
I can't find the collective functions such as:
auto route/descent
hardpoint selection
shkval 7x 23x

And functions like auto-hover, there is the LED exposed, but not the button.

Is there a specific problem? Or if I would find these and add it to the dcs-bios it would get accepted?

**Comment by WarLord211** [comment]
i will look into it

**Comment by truhlikfredy** [comment]
Thanks, going through ka-50 manual and dcs bios reference so far the right panel is there fully. Now going through the back panels, and the left looks good, but I can't find the cockpit aux lamp on the left back side. I can turn on/off all light switches on the whole cockpit, but not the internal aux lamp.

**Comment by WarLord211** [comment]
everything on the Stick has no clickable function so i cant integradt it in BIOS.

Auto Hover is also on the stick? or were can i find it?
cant find the auxlamp. The round one over the light switch?


Doesnt have and know the module. i work with the lua and modell files only. Cant test it ingame

**Comment by truhlikfredy** [comment]
I'm not sure, I was thinking hover is on the collective as I never have seen it directly, but now trying to find it I can't, always I used it as a shortcut and never clicked on it (so then it's non-clickable and unreachable?). I think the door itself is like that as well, but door is not used as often so it doesn't matter.

The cockpit aux lamp (proper name is "Lighting cockpit interior lamp switch") is the only clickable switch on the left side wall behind the pilot. I get you, you did very good job, see attachment:
![lamp](https://user-images.githubusercontent.com/1096073/71382431-b8f8ea00-25cf-11ea-9269-60010bcfd61e.png)

I'm making a integration between nodejs dcs bios and stream deck for ka-50, yesterday I think I finished with all buttons I was intending (except these mentioned in the issues). One more issue I had was with the displays from the R-800-L1 for freq1 and 4, but so far that looks to me as non-DCS BIOS issue as maybe the nodejs module has issue with the double digits. The FREQ2 and 3 work just fine (as they are single digit, in the meantime I got a workaround and calculate the FREQ1 and 4 from the position of the rotation dial itself, so far it works very well).

I was not spending much time on the lamps/lights this time as that I covered previously in a different project and looks like they work just fine.

**Comment by WarLord211** [comment]
Ok i found the lamp and it was not programmed also the switch.
The Door has no clickable programming, so cant convertet in BIOS

(im not the creator of this file, im the keeper ;) )

Update uploaded

**Comment by truhlikfredy** [comment]
Thank you very much for such a quick fix, hope you have great holidays. I got myself Stream Deck week ago and didn't want to use the Win10 only apps, so I used the node.js dcs-bios wrapper. I was in a rush as I wanted to release my project before the end of the holidays so people can enjoy it.  I think I implemented all functional buttons (and the ones which are reachable/clickable). Here is the project: https://github.com/truhlikfredy/dcs-bios-2-stream-deck just in case you want to check it out
